ingest/cell_metadata.py

```diff
@@ -103,6 +114,23 @@ def validate_header_for_coordinate_values(self):
             self.store_validation_issue("error", "format", msg)
             return False
 
+    def conforms_to_metadata_convention(self):
+        """ Determines if cell metadata file follows metadata convention"""
+        convention_file_object = IngestFiles(self.JSON_CONVENTION, ["application/json"])
```

Comment:
Out of scope, but it'd be nice if we define default MIME types for given extensions (e.g. (`"json": ["application/json"]`) in only one place, so we could typically omit second parameters like this `["application/json"]`.